### DISCUSSION TITLE
Should we allow for grey-background table headers/sticky columns?

### DISCUSSION THREAD

**Discussion by jpuzz0** [patternfly/patternfly#6539] [General]
[This design](https://www.figma.com/proto/etbN0ccCcZR1qKCpJXQNAg/Pipeline-phase-2?page-id=615%3A34927&type=design&node-id=1750-107139&viewport=1131%2C-244%2C0.1&t=I8pPcuoDXIu1XQas-1&scaling=scale-down-width&starting-point-node-id=1585%3A95719) from Openshift AI introduces an idea/use-case not yet documented in Patternfly for tables. 

After discussing more with @yannnz & @xianli123 (designers of this Openshift AI feature), we landed on a slight variation to the original design:
<img width="1130" alt="image" src="https://github.com/patternfly/patternfly-react/assets/96431149/96024fce-b5d2-4a6e-b67c-c5a8fd60a318">

Where the grey color being used here is `pf-v5-u-background-color-200`, which differs from the grey color used when applying `isStriped` to `Tr` components.

A different question that could override the title of this discussion would be: Does this conflict with PF design guidelines or desired guidelines for the use-case of comparing data in a table, and if this isn't what PF would recommend for comparison of data, should we formally define this in PF's documentation with guidelines/examples?

**Comment by nicolethoen** [comment]
@andrew-ronaldson @srambach @mattnolting

**Reply by mattnolting** [reply]
Yes, I believe we should. 
* Supports alternative use cases
* Adds flexibility to table cell styling
